examples/json_validation/README.md

```diff
@@ -60,8 +60,8 @@ kind create cluster
 
 istioctl install --set profile=demo -y
 kubectl label namespace default istio-injection=enabled
-kubectl apply -f https://github.com/istio/istio/blob/48cdd8/samples/httpbin/httpbin.yaml
-kubectl apply -f https://github.com/istio/istio/blob/48cdd8/samples/httpbin/httpbin-gateway.yaml
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.12/samples/httpbin/httpbin.yaml
```

Comment:
Interesting. I tried this raw file and did not work so I opened this PR https://github.com/istio/istio/pull/38114, however I am trying now and it works correctly.

Reply:
I'm not sure what might have changed, perhaps you had a typo in your command?